training/main.py

```diff
@@ -0,0 +1,19 @@
+from sqlalchemy import create_engine
+from sqlalchemy.orm import sessionmaker  
+from sqlalchemy.ext.declarative import declarative_base
+
+
+Base = declarative_base()
+
+engine = create_engine('postgresql+psycopg2://postgres:postgres@localhost/baseparaflask')
```

Comment:
This connection string should go on an envirnoment variable or configuration file not tracked by git